Client/Application/AppDelegate.swift

```diff
@@ -260,22 +260,35 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         var url: String?
         var isPrivate: Bool = false
+        var fxaParams: [String: String] = [
+            "view" : "",
+            "email" : "",
+            "code" : ""
+        ]
```

Comment:
For something like this it might be simpler to use a struct. You'll be able to use dot notation and better type checking.